.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -7,3 +7,15 @@ test:
     - cmake --build build -j48
     - cd build
     - ctest -V
+
+INTEL-DATA-CENTER-MAX-1100:
+  stage: test
+  tags: [intel-data-center-max-1100]
+  image: intel/oneapi-basekit:2024.2.1-0-devel-ubuntu22.04
+  script:
+    - sycl-ls
```

Comment:
Not necessary but this helped having a good idea what backends (with names) are available on the machine.